frontend/src/config/myanmar/reports.json

```diff
@@ -0,0 +1,92 @@
+{
+  "flood_extent_report": {
+    "layer_id": "flood_extent",
+    "type": "floods",
+    "title": "Event name: Flood Report",
+    "publication_date_label": "Publication Date",
+    "sub_text": "This is an automated report. Information should be treated as preliminary",
+    "areas_legend_definition": {
+      "title": "Areas",
+      "items": [
+        {
+          "title": "Province",
+          "color": "#000000"
+        },
+        {
+          "title": "District",
+          "color": "#999797"
+        },
+        {
+          "title": "Township",
+          "color": "#D8D6D6"
+        }
+      ]
+    },
+    "type_legend_definition": {
+      "title": "Potential Flooding",
+      "items": [
+        {
+          "title": "Flooded",
+          "color": "#a50f15"
+        }
+      ]
+    },
+    "map_footer_text": "Sources WFP, UNGIWG, OCHA, GAUL, USGS, NASA, UCSB",
+    "map_footer_sub_text": "The designations employed and the presentation of material in the map(s) do not imply the expression of any opinion on the part of WFP concerning the legal or constitutional status of any country, territory, city or sea, or concerning the delimitation of its frontiers or boundaries.",
+    "table_name": "Population Exposure",
+    "signature_text": "PRISM automated report"
```

Comment:
Why `signature_text` is a configurable parameter ? Do we need to change it in the long run ?
Looking at the PR description, I would also ask for other parameters such as `sources_primary_text`, `sources_secondary_text`, `table_name`